frontend/src/card/card.controllers.js

```diff
@@ -891,19 +898,9 @@ CardControllers.controller('CardDetail', [
             }
         });
 
-        // TODO: while we wait for the databases list we should put something on screen
-        // grab our database list, then handle the rest
-        async function loadDatabasesAndTables() {
-            let dbs = await Metabase.db_list().$promise;
-            return await * dbs.map(async function(db) {
-                db.tables = await Metabase.db_tables({ dbId: db.id }).$promise;
-                return db;
-            });
-        }
-
         async function init() {
             try {
-                databases = await loadDatabasesAndTables();
+                databases = await Metabase.db_list_with_tables().$promise;
```

Comment:
the endpoint `/api/database/:id/metadata` provides a complete view of a database that can be used in tools like the QB when a user is interacting with a database.  by providing things like fields, segments, & metrics we avoid the further api calls which would be required to get these things.